packages/opentelemetry-tracing/test/Tracer.test.ts

```diff
@@ -220,4 +222,60 @@ describe('Tracer', () => {
     assert.strictEqual(context.traceFlags, TraceFlags.NONE);
     span.end();
   });
+
+  it('should start an active span with name and function args', () => {
+    const tracer = new Tracer(
+      { name: 'default', version: '0.0.1' },
+      { sampler: new TestSampler() },
+      tracerProvider
+    );
+
+    assert.strictEqual(tracer.startActiveSpan('my-span', span => {
+      try {
+        // assert.strictEqual(getSpan(context.active()), span)
```

Comment:
Why is this commented out?

Reply:
ah I meant it as a todo, will add the string `todo` and do it before mergine

Reply:
todo comment added, currently `getSpan(context.active())` returns `undefined`